### ISSUE TITLE
Pressing 'search' in empty masstagging dialog causes crash

### ISSUE THREAD

**Issue by sandrotosi** [puddletag/puddletag#526]
<a href="https://github.com/blueblots"><img src="https://avatars2.githubusercontent.com/u/63152708?v=4" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [blueblots](https://github.com/blueblots)**
_Fri Jul 31 13:15:17 2020_
_Originally opened as https://github.com/sandrotosi/puddletag/issues/55_

----

A fix for this is in https://github.com/sandrotosi/puddletag/pull/56

1. Open puddletag and open mass tagging panel
2. Click search without selecting a profile.

Puddletag crashes with the exception:

```
puddletag Version: 2.0.0
Locale: en_US
Traceback (most recent call last):
  File "../puddletag/source/puddlestuff/masstag/dialogs.py", line 589, in lookup
    self._start()
  File "../puddletag/source/puddlestuff/masstag/dialogs.py", line 625, in _start
    mtp.file_pattern)
AttributeError: 'NoneType' object has no attribute 'file_pattern'
```

**Comment by blueblots** [comment]
This issue was fixed in #545, this issue should be closed now.

**Comment by sandrotosi** [comment]
fixed via #545
